container/go/cmd/puller/puller.go

```diff
@@ -120,6 +122,56 @@ func pull(imgName, dstPath, format, cachePath string, platform v1.Platform) erro
 		}
 	}
 
+	if err := generateSym(img, dstPath); err != nil {
+		return errors.Wrapf(err, "failed to generate symbolic links to pulled image at %s", dstPath)
+	}
+
+	return nil
+}
+
+// generateSym creates symbolic links to the config.json and .tar.gz layers
```

Comment:
generateSymlinks creates predictable symbolic links for the config.json and layer tar.gz files so that they can be easily consumed by container_import targets.